flexmeasures/api/v3_0/accounts.py

```diff
@@ -192,6 +194,23 @@ def patch(self, account_data: dict, id: int, account: Account):
                 ):
                     return {"errors": ["Invalid consultancy_account_id"]}, 422
 
+        # Validate color values
```

Comment:
This is already done on main, and even in a better place (`AccountSchema`), so something to remove.

Reply:
yeah I will do that and also remove the one for log_url as well